src/utils/component-versioning.js

```diff
@@ -9,24 +13,39 @@ const versionedTag = (tagName, version) => `${tagName}-${toKebabCase(version)}`;
 
 const extractDependencies = componentClass => {
   const { versions, tagName } = componentClass;
-  // extract all dependencies by comparing with master-component tag name
-  const dependencies = Object.keys(versions).filter(name => name !== tagName);
+  // extract all dependencies...
+  const dependencies = Object.keys(versions)
+    // ... by comparing with master-component tag name
+    .filter(name => name !== tagName)
+    // ... and sorting longest-first (e.g. axa-button-link comes before axa-button)
+    .sort((a, b) => b.length - a.length);
   // pair the list of dependency tag names with their versions
   return [dependencies, dependencies.map(_tagName => versions[_tagName])];
 };
 
-const oldTag = (tagName, closing = '') => `<${closing}${tagName}`;
+const oldTag = (tagName, closing = '', openingBracket = '<') =>
```

Comment:
`openingBracket` is a useless parameter, it's never being used outside this method.

Reply:
It is used in line 40, and there are instances using it implicitly (in its default) in line 50 and 55.